### PR TITLE
Custom sink to handle large messages

### PR DESCRIPTION
**Pull request by smahapatra86**
- As part of handling large audit events, we have created a custom sink which will check the message size.

- If the message size is greater than 256 KB it will first push it to a azure blob, get the blob url and push message to event hub with the blob url.

- If message size is less than 256 KB it will be usual flow. It will directly push to event hub.

- We have created a client to use this custom sink. End users will be using this client.

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
   <PropertyGroup>
     <Description>A client library for KMD Logic Audit</Description>
-    <VersionPrefix>1.3.1</VersionPrefix>
+    <VersionPrefix>1.3.2</VersionPrefix>
```
Comment (avin101) [inline Directory.Build.props]: As this is a minor release and not a patch, should we version it to 1.4.0?

Comment (smahapatra86) [reply Directory.Build.props]: > As this is a minor release and not a patch, should we version it to 1.4.0?

yes, makes sense to me. Let me update to 1.4.0

File: Directory.Build.props [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
   <PropertyGroup>
     <Description>A client library for KMD Logic Audit</Description>
-    <VersionPrefix>1.3.1</VersionPrefix>
+    <VersionPrefix>1.4.0</VersionPrefix>
```
Comment (thecorrado) [inline Directory.Build.props:12]: I would bump version to `1.3.2`

Comment (thecorrado) [reply Directory.Build.props:12]: Also update `CHANGES.md` document

**Review by thecorrado — CHANGES_REQUESTED** [review]

File: Directory.Build.props (hunk 2 above)
Comment (thecorrado) [reply Directory.Build.props:12]: Ok, I just saw @avin101 comment and in the end I agree with you guys. So keep `1.4.0` version, but please update document.

Comment (smahapatra86) [reply Directory.Build.props:12]: yes, will update the document.

**Review by thecorrado — APPROVED** [review]
👍

**Pull request merged**